Assignment2/guilledmv-FromJSON-JSONLD.jsonld

```diff
@@ -0,0 +1,39 @@
+{
+	"@context":{
+
+		"xsd": "http://www.w3.org/2001/XMLSchema#",
+		"student": "http://www.assignment2.com/student/",
+		"Name":"student:Name",
+		"Hometown":"student:Hometown",
+		"Age":"student:Age",
+		"AlmaMater":"student:AlmaMater",
+		"Brother":"student:Brother",
+		"Subjects":"student:Subjects",
+		"Name":"Subjects:Name",
```

Comment:
Rename this "Name" attribute to avoid duplicated definitions. You also have a "Name" for the student.